File: exercise/php/day12/src/Gifts/Child.php
<?php

namespace Gifts;

class Child
{
    public string $name;
    public string $behavior;
    public array $wishlist;

    public function __construct(string $name, string $behavior)
    {
        $this->name = $name;
        $this->behavior = $behavior;
        $this->wishlist = [];
    }

    public function setWishlist(Toy $firstChoice, Toy $secondChoice, Toy $thirdChoice): void
    {
        $this->wishlist = [$firstChoice, $secondChoice, $thirdChoice];
    }
}

File: exercise/C#/day12/Gifts/Gifts/Child.cs
namespace Gifts;

public class Child
{
    public string Name { get; }
    public string Behavior { get; }
    public List<Toy> Wishlist { get; private set; }

    public Child(string name, string behavior)
    {
        Name = name;
        Behavior = behavior;
        Wishlist = [];
    }

    public void SetWishList(Toy firstChoice, Toy secondChoice, Toy thirdChoice)
        => Wishlist = [firstChoice, secondChoice, thirdChoice];
}